hphp/runtime/ext/std/ext_std_process.cpp

```diff
@@ -189,6 +189,8 @@ namespace {
 
 struct ShellExecContext final {
   ShellExecContext() {
+    // Use the default handler while exec'ing in a shell,
+    // saving the previous signal action.
     m_sig_handler = signal(SIGCHLD, SIG_DFL);
```

Comment:
Yeah, you can just set the `sa_action` member to `SIG_DFL` and that should work.